apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx

```diff
@@ -253,7 +253,7 @@ function NoteDisplayInner({
         if (netRecv === 0) {
           return `Gas too high to claim`;
         } else if (isOwnSentNote) {
-          return `Cancel this link, reclaiming ${netDollarsReceivedStr}`;
+          return `Cancel this link, recollecting ${netDollarsReceivedStr}`;
```

Comment:
i think "reclaiming" is fine here, recollect means remember

Reply:
done